pkg/host/internal/network/network.go

```diff
@@ -421,3 +424,40 @@ func (n *network) GetPciAddressFromInterfaceName(interfaceName string) (string,
 	log.Log.V(2).Info("GetPciAddressFromInterfaceName(): result", "interface", interfaceName, "pci address", pciAddress)
 	return pciAddress, nil
 }
+
+func (n *network) DiscoverRDMASubsystem() (string, error) {
+	log.Log.Info("DiscoverRDMASubsystem(): retrieving RDMA subsystem mode")
+	subsystem, err := n.netlinkLib.DiscoverRDMASubsystem()
+
+	if err != nil {
+		log.Log.Error(err, "DiscoverRDMASubsystem(): failed to get RDMA subsystem mode")
+		return "", err
+	}
+
+	return subsystem, nil
+}
+
+func (n *network) SetRDMASubsystem(mode string) error {
+	log.Log.Info("SetRDMASubsystem(): Updating RDMA subsystem mode")
+
+	modeValue := 1
+	if mode == "exclusive" {
+		modeValue = 0
+	}
+	config := fmt.Sprintf("options ib_core netns_mode=%d\n", modeValue)
```

Comment:
I think if you use this we can remove the other file in the manifests no?

Reply:
it's deleted now